guillotina/contrib/swagger/services.py

```diff
@@ -91,11 +91,12 @@ def get_endpoints(self, iface_conf, base_path, api_def, tags=None):
         for method in iface_conf.keys():
             if method == "endpoints":
                 for name in iface_conf["endpoints"]:
+                    for http_method in iface_conf["endpoints"][name].keys():
+                        for key in iface_conf["endpoints"][name][http_method].keys():
+                            if key in "tags":
```

Comment:
What's the type of `key`? If it's a string (and we expect the string `tags`), then this conditional should be `key == "tags"` or `key in ("tags",)`?